roles/ansible_version/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: "Fail if ansible-core version is lower than {{ ansible_core_version }}"
+  ansible.builtin.fail:
+    msg: "Ansible core version is {{ ansible_version.full }}, see https://osism.tech/docs/guides/other-guides/testbed/ for required version."
```

Comment:
I think it makes sense to change this to "Ansible core is {{ ansible_version.full }}, expected Ansible core version is {{ ansible_core_version }}.". We'll use this role not only in the testbed in the future.